docs/constitution.rst

```diff
@@ -0,0 +1,60 @@
+Mu's Constitution
+=================
+
+Our Values
+----------
+
+Happily, Mu's users and developers are a diverse and international community
+who maintain a reputation as a friendly, welcoming and dynamic group.
+
+We trust that such collaborators will treat each other in a way that reflects
+the widely help view that diversity and friendliness are strengths of our
+community to be celebrated and fostered.
+
+We believe anyone involved in Mu's community has a right to:
+
+* be treated with courtesy, dignity and respect;
+* be free from any form of discrimination, victimisation, harassment or
+  bullying.
+* enjoy an environment free from unwelcome behaviour, inappropriate language
+  and unsuitable imagery.
+
+To this end, we hope participants in our community abide by the spirit of our
```

Comment:
We hope or we ask/require?

Reply:
For me, I don't like require. It's not very friendly. I think we should "ask and hope", thus setting a clear expectation of the sorts of things we want without being bossy. Of course, we explain lower down in the constitution what we may do if folks don't abide by the spirit... which I hope shows we're very serious about this sort of thing. TL;DR - I want to use language that expresses our openness, friendliness and respect of others so folks see we practice what we preach.

Reply:
Completely agree! "Ask and hope" sounds good.

I also like the fact that we are asking to follow the spirit, too many times people try to get away with stuff by "following the letter".

Reply:
The very slight weakness of this phrasing comes from the fact that in most situations in English the subjunctive takes the same form as the indicative. Here, we'd like to highlight our wish for the future / ongoing behaviour of the community, not merely to express a tentative expression about its present behaviour.

Having made that somewhat pretentious assertion, I can't come up with a better wording at present which doesn't start to read like an Act of Parliament!